docs/ecosystem-riot-testing-library.md

```diff
@@ -0,0 +1,36 @@
+---
+id: ecosystem-riot-testing-library
+title: riot-testing-library
+---
+
+[`riot-testing-library`][gh] builds on top of [DOM Testing Library](https://github.com/testing-library/dom-testing-library) by adding APIs for working with [Riot.js](https://riot.js.org/) components.
+
+```
+npm install --save-dev riot-testing-library
+```
+
+```javascript
+import render, { cleanup, fireEvent } from 'riot-testing-library'
+import TestTag from './test.tag'
+
+afterEach(() => {
+  cleanup()
```

Comment:
Have you considered [making this cleanup automatic](https://testing-library.com/docs/react-testing-library/setup#skipping-auto-cleanup) ([like React Testing Library](https://github.com/testing-library/react-testing-library/blob/89d11b08c61f3cf5af770c61a03d564b3fdc301c/src/index.js#L4-L14))?

Reply:
This feature is awesome !
I don't pay attention to it when I use react-testing-library. It's easy to use.
I add this feature.